packages/web-monetization-wext/src/content/wmPolyfill.ts

```diff
@@ -1,7 +1,8 @@
 const events = [
   'monetizationstart',
   'monetizationstop',
-  'monetizationpending'
+  'monetizationpending',
```

Comment:
need a CI test that checks the manifest.json hash listed in the CSP matches the hashed polyfill